src/components/ATATSummaryCard.vue

```diff
@@ -146,13 +161,28 @@ export default class ATATSummaryCard extends VCard {
     return true;
   }
 
-  @Emit()
   private leftButtonClicked(card: ATATSummaryCardItem) {
-    return true;
+    let id = card.title;
+    switch (this.$route.name) {
+      case "fundingsummary":
+        this.$router.push({ name: "editfunding", params: { id: `${id}` } });
+        break;
+      case "applicationsummary":
+        this.$router.push({
+          name: "editapplication",
+          params: { id: `${id}` },
+        });
+        break;
+      default:
+      // code block
+    }
   }
-
   @Emit()
   private rightButtonClicked(card: ATATSummaryCardItem) {
+    if (card.type === "TASK ORDER") {
```

Comment:
This can stay because the delete modal is on this page. If we need to add another button for non-delete functionality  in the future we will. Right now, tho, our right button is always be delete.